gradle.properties

```diff
@@ -13,4 +13,4 @@ org.gradle.jvmargs=-Xmx2560m -XX:MaxPermSize=256m -Dfile.encoding=UTF-8
 kotlin.code.style=official
 org.gradle.parallel=true
 org.gradle.daemon=true
-org.gradle.caching=true
+org.gradle.caching=false
```

Comment:
why disabling cache?

Reply:
This flag makes command `./gradlew test` unusable, even after `./gradlew clean`.

Reply:
but we will disable cache for whole project.
I think that if there is some issue with `test` task then we should disable cache only for it